docs/spring-boot/app-starter.md

```diff
@@ -0,0 +1,210 @@
+The Symphony BDK for Java provides a _Starter_ module that aims to ease extension app backend developments within a
+[Spring Boot](https://spring.io/projects/spring-boot) application.
+ 
+## Features
+- Configure extension app through `application.yaml`
+- Expose a REST API to perform [Circle of Trust authentication](https://developers.symphony.com/extension/docs/application-authentication).
+
+## Installation
+
+The following listing shows the `pom.xml` file that has to be created when using Maven:
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.example</groupId>
+    <artifactId>bdk-app-spring-boot</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>bdk-app-spring-boot</name>
+    
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.symphony.platformsolutions</groupId>
+                <artifactId>symphony-bdk-bom</artifactId>
+                <version>1.3.2.BETA</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.symphony.platformsolutions</groupId>
+            <artifactId>symphony-bdk-app-spring-boot-starter</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter</artifactId>
+        </dependency>
+    </dependencies>
+    
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.springframework.boot</groupId>
+                    <artifactId>spring-boot-maven-plugin</artifactId>
+                    <version>2.3.4.RELEASE</version>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+    </build>
+</project>
+```
+The following listing shows the `build.gradle` file that has to be created when using Gradle:
+```groovy
+plugins {
+    id 'java-library'
+    id 'org.springframework.boot' version '2.3.4.RELEASE'
+}
+
+dependencies {
+    implementation platform('com.symphony.platformsolutions:symphony-bdk-bom:1.3.2.BETA')
+    
+    implementation 'com.symphony.platformsolutions:symphony-bdk--spring-boot-starter'
+    implementation 'org.springframework.boot:spring-boot-starter'
+}
+```
+
+## Create a Simple Backend for Extension Application
+As a first step, you have to initialize your environment through the Spring Boot `src/main/resources/application.yaml` file: 
+```yaml
+bdk:
```

Comment:
We should have: 
```yaml
bdk:
  host: acme.symphony.com
  bot:
    username: mybot
    privateKeyPath: /path/to/rsa/privatekey.pem
  app:
    appId: myapp
    privateKeyPath: /path/to/rsa/privatekey.pem

bdk-app:
  auth:
    enabled: true # enable the CircleOfTrust controller (default is true)
    jwtCookie:
      enabled: true
      maxAge: 1d # see https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-external-config-conversion-duration
```
⚠️ You should really consider the documentation as also important as the code you write. If the documentation is wrong, developers won't know how to use what you developed. And they might contact the Symphony support for that, leading to a support request opened on our side and finally time consumed for nothing (for us and support guys).

Reply:
Documentation is not something we write just for fun 

![attention](https://media.giphy.com/media/fQGf2FTKayOnKjwidV/giphy.gif)